Blog post announcing the selection of Jenkins as GSoC Mentoring Org. (#6063)
jenkins-infra/jenkins.io: content/blog/2023/02/2023-02-23-gsoc2023-announcement.adoc

We are thrilled to announce that Jenkins has been accepted to 
link:https://summerofcode.withgoogle.com/[Google Summer of Code 2023]! 
This will be Jenkins' seventh year as a mentoring organization. 

Congratulations to all link:https://summerofcode.withgoogle.com/programs/2023/organizations[172 accepted open source participating organizations]!

For the past 6 years as a mentoring organization, 
Jenkins boasts mentorship of 31 GSoC students by 85+ mentors, 
that's bringing together over 107 strangers for a common idea - Jenkins. 

At the heart of it, GSoC is more than just a mentoring program, bridging new contributors to the open source. 
It is about giving a little of your day to make a lifetime of a difference, 
not only for the GSoC contributors but also for the many Jenkins users who will benefit from the improvements.  

We are excited to welcome new GSoC contributors to the Jenkins family. 
We think you will enjoy this valuable experience while developing your technical skills. 
You will gain insights into how the community works. 
The best part will be learning from people who are passionate about Jenkins 

> but more importantly they passionate about wanting to make a difference for another individual (GSoC contributors) and for the betterment of the project as a whole.

Reviewer: ```suggestion
but more importantly they are passionate about wanting to make a difference for another individual (GSoC contributors) and for the betterment of the project as a whole.
```

> For detailed information on Jenkins GSoC 2022, see link:/projects/gsoc/2022/[completed projects].

Reviewer: ```suggestion
For detailed information on Jenkins GSoC 2022, see link:/projects/gsoc/2022/[the completed projects].
```

image:/images/gsoc/2023/GSoC2023-here_we_come.png[Here we come]

**What's next?**
GSoC is link:https://opensource.googleblog.com/2023/02/mentor-organizations-announced-for.html[officially announced], and please expect more potential GSoC contributors to contact projects in our
link:/projects/gsoc#contacts[Gitter channels and Discourse channels].
Many communications will also happen in SIG and sub-project channels.

> We will be working hard in order to help potential participants to find interesting projects, to explore the area,

Reviewer: ```suggestion
We will be working hard in order to help potential participants to find interesting projects, to explore the the relevant domain(s),
```

> and to prepare their project proposals before the deadline on April 4th.

Reviewer: ```suggestion
and to prepare their project proposals before the deadline on April 4th (UTC).
```

Then we will process the applications, select projects and assign mentor teams.

All information about the Jenkins GSoC is available on its link:/projects/gsoc/[sub-project page].

== How do I apply?

See the link:/projects/gsoc/students[Information for students] page for full application guidelines.

We encourage interested participants to reach out to the Jenkins community early and to start exploring project ideas.

> We also encourage participants to join the weekly link:https://docs.google.com/document/d/1UykfAHpPYtSx-r_PQIRikz2QUrX1SG-ySriz20rVmE0/edit?usp=sharing[Jenkins GSoC office hours], these meetings are set up for participants to meet org admins and mentors and to ask questions.

Reviewer 2: ```suggestion
We also encourage participants to join the weekly link:https://docs.google.com/document/d/1UykfAHpPYtSx-r_PQIRikz2QUrX1SG-ySriz20rVmE0/edit?usp=sharing[Jenkins GSoC office hours], as these meetings are set up for participants to meet org admins and mentors, and to ask questions.
```
small suggestion for wording/punctuation

> Also, join our link:https://app.gitter.im/#/room/#jenkinsci_gsoc-sig:gitter.im[Gitter channel] and our
> link:https://community.jenkins.io/c/contributing/gsoc/6[Discourse server] 
> to receive information about such incoming events in the project.

Reviewer 2: ```suggestion
Also, join our link:https://app.gitter.im/#/room/#jenkinsci_gsoc-sig:gitter.im[Gitter channel] and our link:https://community.jenkins.io/c/contributing/gsoc/6[Discourse server] to receive information about such incoming events in the project.
```
small suggestion for formatting

> The application period starts on March 20th, but you should prepare now!

Reviewer: ```suggestion
The application period starts on March 20th (UTC), but you should prepare now!
```

Use the time before the application period to discuss and improve your project proposals.
We also recommend that you become familiar with Jenkins and start exploring your proposal areas.

> Project ideas include quick-start guidelines and reference newbie-friendly issues
> which may help with initial study.

Reviewer 2: ```suggestion
Project ideas include quick-start guidelines and reference newbie-friendly issues, which may help with initial study.
```
small punctuation/formatting suggestion

> If you do not see anything interesting,
> you can link:/projects/gsoc/proposing-project-ideas/[propose your own project idea].

Reviewer 2: ```suggestion
If you do not see anything interesting, you can link:/projects/gsoc/proposing-project-ideas/[propose your own project idea].
```
sentence per line formatting suggestion

== I want to be a mentor. Is it too late?

It's not!

> We are looking for more project ideas and for Jenkins contributors/users
> who are passionate about Jenkins and want to be a mentor.

Reviewer 2: ```suggestion
We are looking for more project ideas and for Jenkins contributors or users who are passionate about Jenkins and want to be a mentor.
```
small wording/formatting suggestion

> No hardcore experience required, mentors can study the project internals together with GSoC contributors and technical advisors.

Reviewer 2: ```suggestion
No hardcore experience is required, as mentors can study the project internals together with GSoC contributors and technical advisors.
```
small wording suggestion

You can either propose a new project idea or join an existing one.

> See the link:/blog/2022/12/09/GSoC-the-gift-of-mentorship//[Call for Mentors post]

Reviewer: ```suggestion
See the link:/blog/2022/12/09/GSoC-the-gift-of-mentorship//[Call for mentors post]
```

> See the link:/blog/2022/12/09/GSoC-the-gift-of-mentorship//[Call for Mentors post]
> and link:/projects/gsoc/mentors[Information for mentors] for details.

Reviewer 2: ```suggestion
Refer to the link:/blog/2022/12/09/GSoC-the-gift-of-mentorship//[call for mentors post] and link:/projects/gsoc/mentors[information for mentors] page for details.
```
small suggestion on formatting/wording

> If you want to propose a new project,
> please do so as soon as possible so that potential GSoC contributors have time to explore them and to prepare their proposals.

Reviewer 2: ```suggestion
If you want to propose a new project, please do so as soon as possible so that potential GSoC contributors have time to explore them and prepare their proposals.
```
small formatting/wording suggestion

This year mentorship does **NOT** require strong expertise in Jenkins development.
The objective is to guide participants and to get involved into the Jenkins community.

> GSoC org admins will help to find advisers if special expertise is required.

Reviewer 2: ```suggestion
GSoC org admins will help to find advisors if special expertise is required.
```
small spelling suggestion

== Important dates for GSoC 2023

> * February 22 - March 19 - Potential GSoC contributors discuss application ideas with mentoring organizations. Show link:https://community.jenkins.io/c/contributing/gsoc/6[us] your proposal!
> * March 20 - GSoC contributor application period begins
> * April 4 - GSoC contributor application deadline
> * May 4 - Accepted GSoC contributor projects announced
> * May 4 - May 28 - Community Bonding Period | GSoC contributors get to know mentors, read documentation, get up to speed to begin working on their projects
> * May 29 - Coding officially begins!
> * July 14 Midterm evaluation
> * August 21 - August 28 - Final week: GSoC contributors submit their final work product
> * September 5 - Initial results of Google Summer of Code 2022
> * November 6 - Final date for all GSoC contributors to submit their final work product

Reviewer 2: ```suggestion
* *February 22 - March 19* - Potential GSoC contributors discuss application ideas with mentoring organizations.
Show link:https://community.jenkins.io/c/contributing/gsoc/6[us] your proposal!
* *March 20* - The GSoC contributor application period begins.
* *April 4* - GSoC contributor application deadline
* *May 4* - Accepted GSoC contributor projects are announced.
* *May 4 - May 28* - Community Bonding Period | GSoC contributors get to know mentors, read documentation, and get up to speed to begin working on their projects.
* *May 29* - Coding officially begins!
* *July 14* - Midterm evaluation
* *August 21 - August 28* - Final week: GSoC contributors submit their final work product.
* *September 5* - Initial results of Google Summer of Code 2022 are published.
* *November 6* - Final date for all GSoC contributors to submit their final work product.
```
adding bold to dates to make them stand out more from the regular text

> See the link:https://developers.google.com/open-source/gsoc/timeline[GSoC Timeline] for more info.

Reviewer 2: ```suggestion
Refer to the link:https://developers.google.com/open-source/gsoc/timeline[GSoC Timeline] for more info.
```
small wording suggestion
